File: An2/MPP/mpp-proiect-java-DeepSmeag/modelModule/src/main/java/contests/model/NumeProba.java
package contests.model;

import contests.model.entity.Entitate;

public class NumeProba extends Entitate<Integer> {
    private String numeProba;
    public NumeProba() {}
    public NumeProba(Integer id, String numeProba){
        super(id);
        this.numeProba = numeProba;
    }
    public String getNumeProba() {
        return numeProba;
    }

    public void setNumeProba(String numeProba) {
        this.numeProba = numeProba;
    }

    public String toString() {
        return numeProba;
    }
}

File: An2/MPP/mpp-proiect-csharp-DeepSmeag/models/NumeProba.cs
using mpp_proiect_csharp_DeepSmeag.models.entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace mpp_proiect_csharp_DeepSmeag.models
{
    [Serializable]
    public class NumeProba : Entitate<int>
    {
        public string numeProba { get; set; }
        public NumeProba() : base() { }
        public NumeProba(int id, String numeProba) : base(id)
        {
            this.numeProba = numeProba;
        }
        override
        public string ToString()
        {
            return numeProba;
        }
    }
}
